components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m

```diff
@@ -81,7 +81,9 @@ - (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibB
                                         @"Trailing Floating Button",
                                         @"Primary Elevation Floating Button",
                                         @"Secondary Elevation Floating Button",
-                                        @"Visible FAB" ];
+                                        @"Visible FAB",
+                                        @"",
+                                        @""]; // the last two empty items are used for offset
```

Comment:
I'm a little confused about what's going on... Is the problem that without these extra cells the last few cells are obscured by the bottom app bar/fab? If this is the case, it might be worth taking advantage of the [`contentInset`](https://developer.apple.com/documentation/uikit/uiscrollview/1619406-contentinset) property to make sure you can scroll to every cell, even the part that the app bar is covering. Not sure though, I could be misunderstanding something.

Reply:
Good point! Addressed in 08dffd2